docs/Information_nodes.rst

```diff
@@ -70,6 +70,60 @@ commands::
 
     --gres=gpu:a100:<number> --reservation=DGXA100
 
+MIG
+^^^
+
+.. _mig_nodes:
+
+MIG (`Multi-Instance GPU <https://www.nvidia.com/en-us/technologies/multi-instance-gpu/>`_)
+is an NVIDIA technology allowing certain GPUs to be
+partitioned into multiple *instances*, each of which has a roughly proportional
+amount of compute resources, device memory and bandwidth to that memory.
+
+NVIDIA supports MIG on its A100 GPUs and allows slicing the A100 into up to 7
+instances. Although this can theoretically be done dynamically, the SLURM job
+scheduler does not support doing so in practice as it does not model
+reconfigurable resources very well. Therefore, the A100s must currently be
+statically partitioned into the required number of instances of every size
+expected to be used.
+
+The new ``cn-g`` series of nodes include A100-80GB GPUs. One third have been
+configured to offer regular (non-MIG mode) ``a100l`` GPUs. The other two thirds
+have been configured in MIG mode, and offer the following profiles:
+
++-----------------------------+----------------------------------------+--------------+
+|          Name               |     GPU                                | Cluster-wide |
+|                             +----------+---------------+-------------+--------------+
+|                             |   Model  |     Memory    |   Compute   |      #       |
++=============================+==========+===============+=============+==============+
+| ``a100l.1g.10gb``:h:`<p>`   |          | 10GB :h:`<p>` | 1/7th       |     72       |
+| ``a100l.1``                 | A100     | (1/8th)       | *of full*   |              |
++-----------------------------+----------+---------------+-------------+--------------+
+| ``a100l.2g.20gb``:h:`<p>`   |          | 20GB :h:`<p>` | 2/7th       |     108      |
+| ``a100l.2``                 | A100     | (2/8th)       | *of full*   |              |
++-----------------------------+----------+---------------+-------------+--------------+
+| ``a100l.3g.40gb``:h:`<p>`   |          | 40GB :h:`<p>` | 3/7th       |     72       |
+| ``a100l.3``                 | A100     | (4/8th)       | *of full*   |              |
++-----------------------------+----------+---------------+-------------+--------------+
+
+And can be requested using a SLURM flag such as ``--gres=gpu:a100l.1``
+
+The partitioning may be revisted as needs and SLURM capabilities evolve. Other
```

Comment:
revised? or revisited?

Reply:
In my head I had "revisited" but with my typos it's exactly Levenshtein edit distance=1 from two alternatives that have the same meaning here. Which would you like?

Reply:
One of both. :stuck_out_tongue_closed_eyes:

Reply:
Revised then. 2 bytes and 1 syllable shorter than the alternative.